AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m

```diff
@@ -85,7 +85,23 @@ - (PINRemoteImageManager *)sharedPINRemoteImageManager
   static PINRemoteImageManager *sharedPINRemoteImageManager = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
+  
 #if PIN_ANIMATED_AVAILABLE
+    // Check that Carthage users have linked both PINRemoteImage & PINCache by testing for one file each
+    if (!(NSClassFromString(@"PINRemoteImageManager.m"))) {
```

Comment:
You should not to add an `.m` in this case ... just use `NSClassFromStirng(@"PINRemoteImageManader")`